common/src/main/java/org/apache/celeborn/common/util/JavaUtils.java

```diff
@@ -418,4 +420,21 @@ public static void shuffleArray(int[] arr, int[] inversed) {
       }
     }
   }
+
+  /**
+   * For jdk8, there is bug for ConcurrentHashMap#computeIfAbsent, checking the key existence to
+   * speed up. See details in CELEBORN-474.
+   */
+  public static <K, V> ConcurrentHashMap<K, V> newConcurrentHashMap() {
+    return new ConcurrentHashMap<K, V>() {
```

Comment:
Will JVM generate a new anonymous class each time?

Reply:
Will check it

Reply:
Since JDK-8161372 got fixed in JDK9, we can define and return a static inner class for JDK8, and simply return a ConcurrentHashMap for JDK9 and above

Reply:
Ok， Will follow up it